### PR TITLE
Add missing definition and instantiation for FETools::get_fe_from_name

### PR DESCRIPTION
**Pull request by masterleinad**
It seems that in the course of #2784 the definition and instantiation for the deprecated FETools::get_fe_from_name have gone missing...

### CODE REVIEW

File: source/fe/fe_tools.cc [hunk 1]
```diff
@@ -2519,7 +2519,12 @@ namespace FETools
   }
 
 
-
+  template <int dim>
+  FiniteElement<dim> *
+  get_fe_from_name (const std::string &parameter_name)
+  {
+    return get_fe_by_name<dim,dim> (parameter_name);
+  }
```
Comment (kronbichler) [inline source/fe/fe_tools.cc]: We usually put three empty lines between functions. Could you adjust it above and below this function? (I know that we are not consistent in the whole `fe_tools.cc`.)

File: include/deal.II/dofs/dof_accessor.templates.h [hunk 2]
```diff
@@ -2308,6 +2308,7 @@ get_mg_dof_indices (const int,
                     std::vector<types::global_dof_index> &dof_indices,
                     const unsigned int fe_index) const
 {
+  (void) dof_indices;
```
Comment (kronbichler) [inline include/deal.II/dofs/dof_accessor.templates.h]: Thanks for fixing this, I didn't see this.

Comment (masterleinad) [inline include/deal.II/dofs/dof_accessor.templates.h]: Me neither ;-)

File: source/fe/fe_tools.cc (hunk 1 above)
Comment (masterleinad) [inline source/fe/fe_tools.cc]: Updated

Comment (kronbichler) [inline source/fe/fe_tools.cc]: Sorry for nit-picking, but isn't this still only two empty lines after the new function?

Comment (masterleinad) [inline source/fe/fe_tools.cc]: You're right... In the latest commit we have three lines between all functions that do not just differ by template specialization.

**Comment by kronbichler** [comment]
Very nice, thanks for fixing up the file. :+1:

File: include/deal.II/dofs/dof_accessor.templates.h (hunk 2 above)
Comment (bangerth) [inline include/deal.II/dofs/dof_accessor.templates.h]: Ah, thanks. My bad. I think I must have only compiled the debug version.

**Pull request merged**